src/_data/styles/fresh.json

```
@@ -0,0 +1,6 @@
+{
+  "title": "Fresh",
+  "author": "Thomas Champion",
+  "stylesheet": "https://github.com/Tombiliby/stylestage-sass/blob/main/public/style.css",
```

Updating to "raw" URL:

```suggestion
  "stylesheet": "https://raw.githubusercontent.com/Tombiliby/stylestage-sass/main/public/style.css",
```